src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -66,6 +66,10 @@
       TODO: add OptimizeAssemblies="lib/netstandard2.0/System.Threading.Tasks.Extensions.dll"
     -->
     <ExpectedDependency Include="System.Threading.Tasks.Extensions" />
+
+    <ExpectedDependency Include="Microsoft.Win32.Registry"/>
+    <ExpectedDependency Include="System.Security.AccessControl"/>
+    <ExpectedDependency Include="System.Security.Principal.Windows"/>
```

Comment:
This is incorrect. We do not own insertion of these assemblies.

Reply:
Thank you, @genlu and @tmat ! Updated with `<_Dependency Remove=.../>`